src/gax/src/response.rs

```diff
@@ -0,0 +1,332 @@
+// Copyright 2025 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! Response types.
+//!
+//! This module contains types related to Google Cloud service responses.
+//! Notably it contains the `Response` type itself. Typically you'll import
+//! this type.
+//!
+//! # Examples
+//!
+//! Inspecting the result of a request
+//!
+//! ```no_run
+//! # use google_cloud_gax::Result;
+//! # use google_cloud_gax::response::Response;
+//! // A type representing a Google Cloud service resource, for example, a
+//! // secret manager secret.
+//! struct Resource {
+//!   // ...
+//! }
+//!
+//! async fn make_google_service_rpc(project_id: &str) -> Result<Response<Resource>> {
+//!   // ...
+//! # panic!()
+//! }
+//!
+//! # tokio_test::block_on(async {
+//! let response = make_google_service_rpc("my-project").await?;
+//! if let Some(date) = response.headers().get("Date") {
+//!     // do something with the date
+//! }
+//! let resource = response.body();
+//! // do something with
+//! # Result::<()>::Ok(()) });
+//! ```
+//!
+//! Creating a response for mocks
+//!
+//! ```
+//! # use google_cloud_gax::Result;
+//! # use google_cloud_gax::response::Response;
+//! // A type representing a Google Cloud service resource, for example, a
+//! // secret manager secret.
+//! struct Resource {
+//!   // ...
+//! }
+//!
+//! fn make_mock_response(body: Resource) -> Result<Response<Resource>> {
+//!     Ok(Response::from(body))
+//! }
+//! ```
+
+/// Represents a Google Cloud service response.
+///
+/// A response from a Google Cloud service consists of a body (potentially the
+/// unit type), and some metadata, currently just headers.
+///
+/// Typically you get a response as the result of making a request via some
+/// client in the Google Cloud client libraries for Rust. You may also create
+/// responses directly when mocking clients for your own tests.
+///
+/// # Examples
```

Comment:
this is a dup of the above section. Remove

Reply:
It is intentional. The generated documentation for the module and the type look too bare otherwise.  I think the duplication is consistent with the API checklist for Rust, and with other crates, e.g.: https://docs.rs/http/1.3.1/http/response/index.html

Reply:
Apologies, I missed that these were different blocks:

https://github.com/googleapis/google-cloud-rust/blob/8cdec5d362548101c81481568dc8d42618aa33ef/src/gax/src/response.rs#L63-L65